primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -546,6 +595,42 @@ where
     E: Pairing<ScalarField = F>,
     F: FftField,
 {
+    #[cfg(not(feature = "seq-fk-23"))]
+    fn compute_h_poly_parallel(
+        prover_param: impl Borrow<UnivariateProverParam<E>>,
+        poly_coeffs: &[E::ScalarField],
+    ) -> Result<GeneralDensePolynomial<E::G1, F>, PCSError> {
+        if poly_coeffs.is_empty() {
+            // Shouldn't we return error here? But test `round_trip` fails if we return
+            // error
+            return Ok(GeneralDensePolynomial::from_coeff_vec(vec![]));
```

Comment:
We can modify the test if needed. But it is nice when everything works on zero-length inputs.